proto/tendermint/types/params.proto

```diff
@@ -13,7 +13,7 @@ message ConsensusParams {
   EvidenceParams  evidence  = 2;
   ValidatorParams validator = 3;
   VersionParams   version   = 4;
-  TimingParams    timing    = 5;
+  SynchronyParams timing    = 5;
```

Comment:
Should we also update the field name as well?

Both are breaking changes to generated code, but neither is wire-breaking.

Reply:
Ah, thanks for the catch.
The break is in unreleased code, so the change shouldn't truly be breaking at all 🤷

Reply:
Yeah, but it will become a migration guide item later. 🙂